source/globalCommands.py

```diff
@@ -264,6 +265,26 @@ def script_reportCurrentSelection(self,gesture):
 		else:
 			speech.speakTextSelected(info.text)
 
+	def is_system_clock_seconds_visible(self) -> bool:

```

Comment:
please use lowerCamelCase for function names

```suggestion
	def _isSystemClockSecondsVisible(self) -> bool:
```

Reply:
this function should probably be moved to `winAPI` or `systemUtils`